presentation-api/controlling-ua/defaultRequest_error-manual.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Presentation API, testing to start a new presentation with an default request setup. (error - manual)</title>
+<link rel="author" title="Marius Wessel" href="http://www.fokus.fraunhofer.de">
+<link rel="help" href="http://w3c.github.io/presentation-api/#dfn-controlling-user-agent">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<p>Click the button below and abort the selection for an device (close the dialog), to start the manual test.</p>
+<button id="presentBtn" onclick="startPresentation()">Start Presentation Test</button>
+
+<script>
+
+    // -------------------
+    // defaultRequest init
+    // -------------------
+    var validUnixDate = new Date().getTime() + String(Math.floor(Math.random() * 1e5)),
+            validPresURL = '../receiving-ua/idlharness.html#__castAppId__=2334D33A/__castClientId__=' + validUnixDate;
+
+    navigator.presentation.defaultRequest = new PresentationRequest(validPresURL);
+    // ------------------------------
+    // Start New Presentation with
+    // 'default request' Test - BEGIN
+    // ------------------------------
+    var startPresentation = function () {
+        promise_test(function () {
+            return navigator.presentation.defaultRequest.start()
+                    .catch(function (err) {
+
+                        test(function () {
```

Comment:
Please don't make new tests embedded since you can make all the test assertions in one test.